# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../metamail/70_mm-2.7-sendmail.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur mm2.7.orig/src/bin/showexternal mm2.7/src/bin/showexternal
--- mm2.7.orig/src/bin/showexternal	2003-05-14 23:58:27.000000000 +0200
+++ mm2.7/src/bin/showexternal	2003-05-14 23:59:21.000000000 +0200
@@ -219,7 +219,7 @@
 	echo "" >> "$NEWNAME"
 	sed -e 1,/^\$/d < $bodyfile >> "$NEWNAME"
 	echo "" >> "$NEWNAME"
-	/usr/lib/sendmail -t  < "$NEWNAME"
+	/usr/sbin/sendmail -t  < "$NEWNAME"
 	if ($status) then
 	    echo sendmail failed
 	    cd ${METAMAIL_TMPDIR}
